### PR TITLE
Freshen up UI

### PR DESCRIPTION
**Pull request by skidding**
Highlights:
- Blue colors (more cosmosy)
- New fonts
- New scalable SVG icons (no more confusing "play button")
- Improved nav readability (added folder to components)
- Simplified bg
- Improved design of welcome message, including Cosmos mascot and nice fade in transition

fresh-cosmos

### CODE REVIEW

File: packages/react-component-playground/src/components/display-screen.less [hunk 1]
```diff
@@ -5,9 +5,9 @@
 @twinkling-texture: url('../static/twinkling.png');
```
Comment (valentin-radulescu-hs) [inline packages/react-component-playground/src/components/display-screen.less]: Might as well get rid of `twinkling-texture` and `clouds-texture` variables and the corresponding files.

File: packages/react-component-playground/src/components/common.less [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
.../FzRobhkY0s6OxWiNZGmUsCpICavcZh2aXOvxlYMnI2DhDmhfi35xGid7pWLEpjQFgyOKhUiO5CJvqVhZTnI1XQDHZMAKGhNQrg5kHlkB/hysICvSsDBCgU2AFGW8O6dvzMCo3qzsSV59yy7QdHUg7qYLNDMPxpzVkT44yPuXR8wYcdFVpos3IDNNlSFFqaWDscI0yZ9pxdphsKcqC/ay2d2mYC803RqqyNWYO+xfUrVfuoePWO9iTeO8iyy9nSvZdDtXLAeWCfSUgAtKCNZHjv9v5wRgHI4lMEbPoCVIC7ljfQ4naIdr4JSmepQJV2+pIU46trssUdnQlbnupbjmlsLM1cSsHqmyW4naGyorTvuzIpQVJ/P+ivekZoleKI/lgUOqxi/F2XQqMsJoY50GHA6if1h2cCV/moJ9JMNw/xOG+2TqNUhGI5iikoagfdyHE/gSFhbDfiZRtnaGmxoSRPYyIx9qQHNtUqO6js6N7cfujR+rnzCdGRyf4/PSBmq8vg81oVabm6S5Xtli2Te+9G2BeY/ZbdJ6nHlDgRXmBLwUA7Mr7F97QazGif9tf9s/+jmliUq5AtRJaIEixnEmMTYiVdzLAhdKQ5eXNmTT5SVw/2cR8Z/Q/akyiTP3g0bg5k58Iq3/464I05aWP0gKPsUomnuVUboz5iMMM7UIu1ohLk6W2ibTV0mKTd5ceoEgzudokNuWx4+UV0HWrhdF+W7Fu5TTG5TVJcrwUzDZSawyODoGmyXDMOphWaGHtS+Dl9DJ4uWDspewsgc7UQQrZ42liYxaZWkOjsiYNULSCA1pAwHQG0IZe5bUJhPQUZ9AF3R6ktVkDDnKFNDzGxZDczKPWXna7G7h1L7pCvgGqjDwp+tCmwRoeoU0p5S+9EsUG+JJaoEnegvQkvgypNkYaMGSQWQT5dYBQpFbKHOftVzALb7unCr1TIUJ9Na1bbdh/E1ol8f7KOs7/dhw9/BpbloMF9t5TBzpPjv2rk7COMan/2aGHNAc95t5yMXGdhj5Dv3VNhhOfjwbt+/eeT1/qlfkxVnXbZ7fvAtOet7HtJohNyEPhBFedwEt+dk7mRejPDwO6ckWTW8PL9+JNpaLU8D+KkxCT/TUDWh7Qy3lYAMG0AWmDiPcncZ57sry0I14yzw+eOFygfZd9wRSS0f4UfAj+/ryrF7dx8zuoScWo0HNJjd
```
Comment (valentin-radulescu-hs) [inline packages/react-component-playground/src/components/common.less]: How about moving the font to `/static` and loading it through `url-loader` (already used for `.png`)?

File: packages/react-component-playground/src/components/welcome-screen.jsx [hunk 3]
```diff
@@ -3,6 +3,14 @@ import StarryBackground from './starry-background';
...
...vgX4+8AzfBEdki+yQ0rhknH6ayvn11+9fHbjZ8f9ZKGQo6yEMVEwDkS2v3C1xtyq/3SNDeKtcjXnEEoWHktsCnPbhbAJI3PV55gIj5c3sWdx7n/B8SbgXXwRHvKL8UVl8ooLaZz+y/XF3qtXz/X+n9FGvJx7i4mlbeAkSdKqThpu+mqLLXKuejy+jAFWrlJal0zVqjG5ipFsR/arZF1UNSyq1uY2FwHZcQLn/gWO1+D4d1xCqvF/GNZlDEwq+dxwY/y/Xnhu+TXrC4P/c7gxPl8k/hkWNezPMepPF7nQ1cz+VvL3KwXDUWFQdZkZa7VcE1f+t02Qm5C82hr/qoGjj2HdP3XWvNI593NcRk3vi+HQfAkcfkuFeCaN058cbYz/XU/yfY1u+I6ope8dj+cY9OczOcqrrhuuMK8pBdCElKhQ15cqTGKdtT3WruKwXC3vcpfs2TiLc+/Hud/GuT9BMPjCyjd8GRpW1cAQLCZj+3Pj/sYvR93uq9ZX+KGgMXy9CvQeISXbbYe9lmp1K2cjtUTqCWB0y5A3ER4nwIdNWFZpcM/i7Ludc7+N49PO8SV56C/FF50BrANn+GA86H9wtLF6Y9ho34fkO4Nm95UCdpCJzTprru8bFgbZRjKK4YlqnuWKXYHVn6kPT1SwK+ecxXHWWvtecH+Gcx/FuZUvsBbWV6Zh1TEwgZTyuE3i48amvxsPBnuEkG8LW53bTJq8VYXNFwmvzF+Ko12JG9iqsBOgooBiUtbWPZJzk41kN0EzLm635oz9pHPuo9j0U9a4j4Bb48vo0F82f0nuoYw9jxD/xcRjRr31bmNKzo77618TNFtfjdC3O2fvEkIpKvScmvryltlXJnyiFTJQhRKzK/KkzT3nQjbSWawxC1j3jLP2UzZN3o+Un3LGLCBIv9Q905e/YW1taBvAhkvTX8e5X48H601rzO1hq9tNR8M7rTVvFDo4IHXUxpo5Z+y8g4arbECtfuY6CjKYwTMwnLE4Y3DWjJxJh865NaxddnF81qTJcax7EC2PuiS9iHPHkVss8fkyPfRXxF9Z7s0ZCiEeRghMEj9grflLheiioyAdD+4wcXInUu1CiB0C2XE45RBCCK+8JnW268YkfWfSsUtNapJk2ZlkwSbJAkI8q8LweefchjNmFWsvCh3ylXj8/wcAQe0h88Y1RyMAAAAASUVORK5CYII=
```
Comment (valentin-radulescu-hs) [inline packages/react-component-playground/src/components/welcome-screen.jsx]: Similar to fonts, this can be loaded with `url-loader` as well.

**Comment by flaviusone** [comment]
> Note: Boy, I sure do wish we could have a darker theme for the fixture editor so it integrates better with the whole Cosmos look and feel. Not sure how easy/possible it it at this point, good to keep in mind for the future.

You can do it by changing the codemirror theme [here](https://github.com/react-cosmos/react-cosmos/blob/fa57e6ab00f6740fdefa4d40ef8e8d5d58fef505/packages/react-component-playground/src/components/component-playground.jsx#L160) with something like [this](https://codemirror.net/demo/theme.html#material) but it would be better to have that as an optional setting. I personally don't always like dark editor themes.

**Comment by valentin-radulescu-hs** [comment]
Even better if it could be a config option 👍

**Comment by skidding** [comment]
@RadValentin addressed comments!

Customizable theme is a good idea. Another freebie! :)

File: test/karma.conf.js [hunk 4]
```diff
@@ -54,7 +54,7 @@ module.exports = function setKarmaConfig(config) {
             'css-loader',
           ],
         }, {
-          test: /\.png$/,
```
Comment (valentin-radulescu-hs) [inline test/karma.conf.js]: `/\.(png| woff)$/`?

**Comment by skidding** [comment]
Published under `v2.0.0-beta.11`.

**Pull request merged**